utils/iscn.ts

```diff
@@ -55,7 +54,7 @@ export function formatISCNTxPayload (payload: ISCNRegisterPayload): ISCNSignPayl
     usageInfo: license,
     contentFingerprints: [...new Set(contentFingerprints)],
     recordNotes
-  }
+  }as import('@likecoin/iscn-js').ISCNSignPayload
```

Comment:
We can probably use `import { type ISCNSignPayload } from '@likecoin/iscn-js'` at the header, such that the library is only imported for type checking

Reply:
[♻️ Refactor type imports](https://github.com/likecoin/nft-book-press/pull/321/commits/33737c09594920c1859109a109e3413098391f99)